### DISCUSSION TITLE
[Ask] How to use Multi Caret feature?

### DISCUSSION THREAD

**Discussion by sonswift** [AvaloniaUI/AvaloniaEdit#279] [Q&A]
Hi all, I could not use the feature `Multi caret` and I didn't find any place to enable it in code. Anyone can help me? Thanks

**Comment by danipen** [comment]
Multi caret is not fully implemented in AvaloniEdit however multiple edition is available using the rectangular selection that can be done using alt + mouse.

Other ways/commands to get more than one caret would need to be implemented.
